docs/ref/migration-operations.txt

```diff
@@ -269,7 +269,7 @@ be removed (elided) when :ref:`squashing migrations <migration-squashing>`.
 RunPython
 ---------
 
-.. class:: RunPython(code, reverse_code=None, atomic=True, hints=None, elidable=False)
+.. class:: RunPython(code, reverse_code=None, atomic=None, hints=None, elidable=False)
```

Comment:
Shouldn't "(the ``atomic`` parameter has no effect on these databases)" be removed?